package.json

```diff
@@ -90,13 +90,17 @@
     "test:debug": "craco --inspect-brk test --runInBand --no-cache --transformIgnorePatterns 'node_modules/(?!@polkadot)/'",
     "test:e2e-ci": "CI=true npx playwright test /e2e-tests/tests",
     "test:e2e-local": "CI=false npx playwright test /e2e-tests/tests",
+    "test:storybook-test-runner": "test-storybook",
+    "test:playwright:stories": "npx playwright test --config=storybook-testing/playwright.config.ts",
+    "test:stories": "yarn test:storybook-test-runner && yarn test:playwright:stories",
+    "test:stories:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"yarn start-storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"wait-on tcp:6006 && yarn test:stories\"",
     "eject": "craco eject",
     "prettier": "prettier **/** --check --ignore-path .gitignore",
     "lint": "eslint src --max-warnings=0",
     "commit": "git-cz",
     "preci": "rm -fr node_modules",
     "ci": "yarn install --frozen-lockfile",
-    "storybook": "start-storybook -p 6006 -s public",
+    "start-storybook": "start-storybook -p 6006 -s public",
```

Comment:
i'd suggest we do `storybook:start` and `storybook:build`.

Reply:
Thanks @maht0rz take a look at it now